static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -698,6 +686,80 @@ class SpanTree extends Component<PropType> {
     };
   }
 
+  addSpanRowToState = (
+    spanId: string,
+    spanRow: React.RefObject<HTMLDivElement>,
+    treeDepth: number
+  ) => {
+    this.setState((prevState: StateType) => {
+      const newSpanRows = clone(prevState.spanRows);
```

Comment:
nit: you can also use the spread operator to get a new reference

Reply:
yeah you're right why in the world did I use clone here lmao